apps/meeting/src/containers/DynamicMeetingControls/index.tsx

```diff
@@ -36,25 +42,18 @@ const DynamicMeetingControls = () => {
 
   return (
     <StyledControls className="controls" $active={!!isUserActive}>
-      <ControlBar
-        className="controls-menu"
-        layout="undocked-horizontal"
-        showLabels
-      >
-        <ControlBarButton
-          className="mobile-toggle"
-          icon={<Dots />}
-          onClick={handleToggle}
-          label="Menu"
-        />
-        {status === DeviceLabelTriggerStatus.GRANTED ?
+      <ControlBar className="controls-menu" layout="undocked-horizontal" showLabels>
+        <ControlBarButton className="mobile-toggle" icon={<Dots />} onClick={handleToggle} label="Menu" />
+        {status === DeviceLabelTriggerStatus.GRANTED ? (
           <>
-            <AudioInputControl />
-            <VideoInputControl />
+            {isVoiceFocusEnabled ? <AudioInputVFControl /> : <AudioInputControl />}
+            {videoTransformsEnabled ? <VideoInputTransformControl /> : <VideoInputControl />}
```

Comment:
This is to support VF and BG filters in `DynamicMeetingControls`(used for spectator mode which skip device permission when joining the meeting) to align with `MeetingControls`(default one).